examples/load_balancer/load_balancer.c

```diff
@@ -456,6 +456,7 @@ table_lookup_entry(struct rte_mbuf *pkt, struct flow_info **flow) {
                 return -1;
         }
 
+        memset(&key, 0, sizeof(struct onvm_ft_ipv4_5tuple));
```

Comment:
try putting these memsets back inside the fill_key functions and see if there is still a problem.